error/s2n_errno.c

```diff
@@ -263,6 +263,7 @@ static const char *no_such_error = "Internal s2n error";
     ERR_ENTRY(S2N_ERR_INVALID_X509_EXTENSION_TYPE, "Invalid X509 extension type") \
     ERR_ENTRY(S2N_ERR_INSUFFICIENT_MEM_SIZE, "The provided buffer size is not large enough to contain the output data. Try increasing the allocation size.") \
     ERR_ENTRY(S2N_ERR_KEYING_MATERIAL_EXPIRED, "The lifetime of the connection keying material has exceeded the limit. Perform a new full handshake.") \
+    ERR_ENTRY(S2N_ERR_EARLY_DATA_TRIAL_DECRYT, "Unable to decrypt rejected early data") \
```

Comment:
Nit: Typo: `DECRYT` -> `DECRYPT`. Also update the description to include `trial decrypt` terminology for consistency.

Reply:
I prefer the current message. "Unable to decrypt rejected early data" is what "err early data trial decrypt" really expands to.